modules/avatar/react/stories/stories_visualTesting.tsx

```diff
@@ -19,7 +19,7 @@ export default withSnapshotsEnabled({
   component: Avatar,
   parameters: {
     chromatic: {
```

Comment:
We could manually type this [here](https://github.com/Workday/canvas-kit/blob/master/utils/storybook/withSnapshotsEnabled.ts#L6) if we want to avoid issues like this in the future.

Reply:
Yeah. Chances are high that `withSnapshotsEnabled` will have `chromatic` and other stories won't.